rust-runtime/aws-smithy-http-server/examples/README.md

```diff
@@ -0,0 +1,25 @@
+# Smithy Rust Server SDK example
+
+This folder contains an example service called Pokémon Service used to showcase the service framework capabilities and to run benchmarks.
+
+## Build
+
+Since this example requires both the server and client SDK to be code-generated from their [model](https://github.com/awslabs/smithy-rs/tree/main/codegen-server-test/model/pokemon.smithy), a Makefile is provided to build and run the service. Just run `make` to prepare the first build.
```

Comment:
```suggestion
Since this example requires both the server and client SDK to be code-generated from their [model](/codegen-server-test/model/pokemon.smithy), a Makefile is provided to build and run the service. Just run `make` to prepare the first build.
```

GitHub supports relative links. When the link starts with a `/`, it is relative to the root of the repository